File: shared/schemas/health.py
"""Health check schemas."""

from pydantic import BaseModel


class HealthResponse(BaseModel):
    """Health check response."""

    status: str


class ReadinessResponse(BaseModel):
    """Readiness check response."""

    status: str


class RootResponse(BaseModel):
    """Root endpoint response."""

    message: str
    version: str

File: shared/types/health.ts
/**
 * Health check types - TypeScript equivalents of Python Pydantic models
 * Keep in sync with shared/schemas/health.py
 */

export interface HealthResponse {
  status: string;
}

export interface ReadinessResponse {
  status: string;
}

export interface RootResponse {
  message: string;
  version: string;
}
